### ISSUE TITLE
Question: Any idea if this will also work on the NanoPi Zero 2?

### ISSUE THREAD

**Issue by Pauligrinder** [norman-arch/NanoPi-NEO3#3]
Hey, long time no see! 

My NanoPi Neo 3 has been running flawlessly with this bootloader since we managed to solve the issue I had ~3 years ago :) 

Anyway, I recently ordered a NanoPi Zero 2, which I'm planning on replacing my Home-Assistant device with. It's currently running a Raspberry Pi 2 with a USB connected SSD to speed things up, but it's still pretty slow - I imagine the Zero 2 should be exponentially faster, given it has twice the CPU cores (and probably faster cores too), 2GB LPDDR4 RAM and I also chose the 64GB eMMC module that I'll use for storage instead of the usb-ssd. 

So anyway, got any idea whether this bootloader will work on that device too? It's not the exact same Rockchip, but seems very similar in pretty much every way. 
If you don't expect it to work, can you point me in the right direction regarding what to do to get it to run?

**Comment by Pauligrinder** [comment]
I just received my new NanoPi Zero 2, and will try to get around to checking it out tomorrow! I'm hoping it'll be able to output picture through the USB-C port using the same docks I'm using for my Mac, so debugging should be a tad easier than on the Neo 3 which just simply doesn't have any way to output any video. 
According to some pages I found this one doesn't have any video output either, but according to the official documentation it does have a GPU chip... So I don't know 🤣

**Comment by Pauligrinder** [comment]
I gave up. I'll look into getting Arch running on the machine some other time when I've got more time to tinker... For now I settled on the friendlyelec provided Debian version, which seems passable. 
I don't like how out of date the repos are, mainly Python on this Debian version is right on the border of being unusable. I found a 3rd party repo that allowed me to get Python 3.12, but even support for that is already deprecated in Home-Assistant, so I'll have to do something about it if I intend on going past version 2025.2.

But yeah, I thought I would just keep the bootloader from this Debian installation and replace the rootfs with Arch, but nope, I can't find it! I checked the mount points, and for the rootfs it just says "overlay". Whatever that means! Seems like very typical Debian/Ubuntu style - they always want to hide everything under a pretty cover like this, which then makes editing anything a pain in the ass. Also I hate that apt install always immediately enables installed services. I wanna configure it first, and maybe I just want to test something and not have it immediately run on every boot?

So looks like I'm gonna have to follow the friendlyelec wiki guide on building a custom Linux image, and even that guide is annoyingly specific to Debian/Ubuntu... Maybe then I'll make a repo of my own with some scripts/etc to build an Arch image for this particular SBC (which can then be extended to other NanoPi variants, maybe).
